pkg/admin/connector.go

```diff
@@ -169,8 +169,10 @@ func NewConnector(config ConnectorConfig) (*Connector, error) {
 			SASL: mechanismClient,
 			TLS:  tlsConfig,
 			// Hotfix for https://github.com/efcloud/topicctl/issues/112
-			MetadataTTL: 10 * time.Minute,
+			MetadataTTL: defaultTimeout,
+			DialTimeout: defaultTimeout,
 		},
+		Timeout: defaultTimeout,
```

Comment:
What's the value for `defaultTimeout`? Is it not gonna just roll back the original fix to MetadataTTL?